src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines. Please remove the unnecessary empty lines between the package declaration and the class definition.